### PR TITLE
Internal change

### PR DESCRIPTION
**Pull request by jay-shi**
Internal change

### CODE REVIEW

**Comment by jay-shi** [comment]
This is just a test commit to test copybara service. Do not approve.

File: proto/google/events/cloud/audit/v1/data.proto [hunk 1]
```diff
@@ -22,9 +22,6 @@ import "google/protobuf/timestamp.proto";
 import "google/rpc/context/attribute_context.proto";
 import "google/rpc/status.proto";
 
-option csharp_namespace = "Google.Events.Protobuf.Cloud.Audit.V1";
```
Comment (grant) [inline proto/google/events/cloud/audit/v1/data.proto:25]: We can't remove this or the generator will break

**Review by grant — CHANGES_REQUESTED** [review]

**Review by grant — APPROVED** [review]

File: proto/google/events/cloud/audit/v1/data.proto (hunk 1 above)
Comment (grant) [reply proto/google/events/cloud/audit/v1/data.proto]: Comment looks outdated.

**Comment by jay-shi** [comment]
It's just a test PR.

**Pull request closed without merging**